.github/workflows/pypi.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Python distributions to PyPI
+
+on: [push]
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install package deps
+        run: >-
+          sudo apt-get install -y
+          git
+          tar
+          make
+          gcc
+          python3-dev
+          python3-wheel
+          python3-build
+          python3-setuptools
+          python3-pip
+          python3-libvirt
+          python3-sphinx
+          python3-sphinx-rtd-theme
+      - name: Install dependencies
+        run: pip3 install .[lint,test,docs]
+      - name: Fetch date for version bump
+        run: echo "new_version=$(date +'%Y%m%d%H%M')" >> $GITHUB_ENV
+      - name: Replace version in setup.py
+        if: startsWith(github.ref, 'refs/tags/v') != true && github.ref == 'refs/heads/master'
+        # Only replace the version if we are not on a tag and on master.
+        run: sed -i '/__version__ = "[0-9].[0-9].[0-9]/s/.$/.'${{ env.new_version }}'"/g' koan/__init__.py
+      - name: Build a binary wheel and a source tarball
+        run: make release
+      - name: Publish distribution to Test PyPI
+        if: github.ref == 'refs/heads/master'
```

Comment:
Another master reference that should be main

Reply:
Many thanks. That would have been overlooked.